cost-analyzer/values.yaml

```diff
@@ -202,6 +202,50 @@ kubecost:
     #  cpu: "100m"
     #  memory: "256Mi"
 
+# Kubecost Metrics deploys a separate pod which will emit kubernetes specific metrics required 
+# by the cost-model. This pod is designed to remain active and decoupled from the cost-model itself. 
+# However, disabling this service/pod deployment will flag the cost-model to emit the metrics instead. 
+kubecostMetrics: 
+  # emitPodAnnotations: false 
+  # emitNamespaceAnnotations: false 
+  # emitKsmV1Metrics: true
+  
+  # Optional
+  # The metrics emitter is a separate deployment and service (for prometheus scrape auto-discovery)
+  # which emits metrics cost-model relies on. Enabling this deployment also removes the KSM dependency
+  # from the cost-model. If the deployment is not enabled, the metrics will continue to be emitted from
+  # the cost-model.
+  emitter:
+    enabled: false
+    image: "gcr.io/kubecost1/kube-metrics:v0.0.1"
```

Comment:
I'd be reluctant to introduce another name, i.e. "kube-metrics" isn't this the "kubecost_exporter" that has a "kube-metrics" or "ksm" module?

Reply:
Not really, trying avoid name conflict here until we have parity between what we currently call the "exporter" (headless cost-model).